Recent doc updates break the Julia build

Recent doc updates are breaking the Julia build when we incorporate the latest SparseArrays in https://github.com/JuliaLang/julia/pull/45775

https://build.julialang.org/#/builders/54/builds/6998

```
��� Error: duplicate docs found for 'SparseArrays.transpose!' in `@docs` block in src/stdlib/SparseArrays.md:204-236
��� ```@docs
��� SparseArrays.AbstractSparseArray
��� SparseArrays.AbstractSparseVector
��� SparseArrays.AbstractSparseMatrix
��� SparseArrays.SparseVector
��� SparseArrays.SparseMatrixCSC
��� SparseArrays.sparse
��� SparseArrays.sparsevec
��� SparseArrays.issparse
��� SparseArrays.nnz
��� SparseArrays.findnz
��� SparseArrays.spzeros
��� SparseArrays.spdiagm
��� SparseArrays.sparse_hcat
��� SparseArrays.sparse_vcat
��� SparseArrays.sparse_hvcat
��� SparseArrays.blockdiag
��� SparseArrays.sprand
��� SparseArrays.sprandn
��� SparseArrays.nonzeros
��� SparseArrays.rowvals
��� SparseArrays.nzrange
��� SparseArrays.droptol!
��� SparseArrays.dropzeros!
��� SparseArrays.dropzeros
��� SparseArrays.permute
��� permute!{Tv, Ti, Tp <: Integer, Tq <: Integer}(::SparseMatrixCSC{Tv,Ti}, ::SparseMatrixCSC{Tv,Ti}, ::AbstractArray{Tp,1}, ::AbstractArray{Tq,1})
��� SparseArrays.transpose!
��� SparseArrays.adjoint!
��� SparseArrays.halfperm!
��� SparseArrays.ftranspose!
��� SparseArrays.fkeep!
��� ```
��� @ Documenter.Expanders /buildworker/worker/doctest_linux64/build/doc/deps/packages/Documenter/tBpqI/src/Utilities/Utilities.jl:32
��� Error: duplicate docs found for 'SparseArrays.adjoint!' in `@docs` block in src/stdlib/SparseArrays.md:204-236
��� ```@docs
��� SparseArrays.AbstractSparseArray
��� SparseArrays.AbstractSparseVector
��� SparseArrays.AbstractSparseMatrix
��� SparseArrays.SparseVector
��� SparseArrays.SparseMatrixCSC
��� SparseArrays.sparse
��� SparseArrays.sparsevec
��� SparseArrays.issparse
��� SparseArrays.nnz
��� SparseArrays.findnz
��� SparseArrays.spzeros
��� SparseArrays.spdiagm
��� SparseArrays.sparse_hcat
��� SparseArrays.sparse_vcat
��� SparseArrays.sparse_hvcat
��� SparseArrays.blockdiag
��� SparseArrays.sprand
��� SparseArrays.sprandn
��� SparseArrays.nonzeros
��� SparseArrays.rowvals
��� SparseArrays.nzrange
��� SparseArrays.droptol!
��� SparseArrays.dropzeros!
��� SparseArrays.dropzeros
��� SparseArrays.permute
��� permute!{Tv, Ti, Tp <: Integer, Tq <: Integer}(::SparseMatrixCSC{Tv,Ti}, ::SparseMatrixCSC{Tv,Ti}, ::AbstractArray{Tp,1}, ::AbstractArray{Tq,1})
��� SparseArrays.transpose!
��� SparseArrays.adjoint!
��� SparseArrays.halfperm!
��� SparseArrays.ftranspose!
��� SparseArrays.fkeep!
��� ```
��� @ Documenter.Expanders /buildworker/worker/doctest_linux64/build/doc/deps/packages/Documenter/tBpqI/src/Utilities/Utilities.jl:32
[ Info: CrossReferences: building cross-references.
[ Info: CheckDocument: running document checks.
[ Info: Populate: populating indices.
ERROR: LoadError: `makedocs` encountered an error (:docs_block). Terminating build before rendering.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:35
 [2] runner(#unused#::Type{Documenter.Builder.RenderDocument}, doc::Documenter.Documents.Document)
   @ Documenter.Builder /buildworker/worker/doctest_linux64/build/doc/deps/packages/Documenter/tBpqI/src/Builder.jl:255
 [3] dispatch(#unused#::Type{Documenter.Builder.DocumentPipeline}, x::Documenter.Documents.Document)
   @ Documenter.Utilities.Selectors /buildworker/worker/doctest_linux64/build/doc/deps/packages/Documenter/tBpqI/src/Utilities/Selectors.jl:170
 [4] #2
   @ /buildworker/worker/doctest_linux64/build/doc/deps/packages/Documenter/tBpqI/src/Documenter.jl:267 [inlined]
 [5] cd(f::Documenter.var"#2#3"{Documenter.Documents.Document}, dir::String)
   @ Base.Filesystem ./file.jl:112
 [6] makedocs(; debug::Bool, format::Documenter.Writers.HTMLWriter.HTML, kwargs::Base.Pairs{Symbol, Any, NTuple{11, Symbol}, NamedTuple{(:build, :modules, :clean, :doctest, :linkcheck, :linkcheck_ignore, :strict, :checkdocs, :sitename, :authors, :pages), Tuple{String, Vector{Module}, Bool, Bool, Bool, Vector{String}, Bool, Symbol, String, String, Vector{Any}}}})
   @ Documenter /buildworker/worker/doctest_linux64/build/doc/deps/packages/Documenter/tBpqI/src/Documenter.jl:266
 [7] top-level scope
   @ /buildworker/worker/doctest_linux64/build/doc/make.jl:296
in expression starting at /buildworker/worker/doctest_linux64/build/doc/make.jl:296
make: *** [html] Error 1
make: Leaving directory `/buildworker/worker/doctest_linux64/build/doc'
program finished with exit code 2
elapsedTime=305.802036
```

--- a/docs/src/index.md
+++ b/docs/src/index.md
@@ -228,8 +228,6 @@ SparseArrays.dropzeros!
 SparseArrays.dropzeros
 SparseArrays.permute
 permute!{Tv, Ti, Tp <: Integer, Tq <: Integer}(::SparseMatrixCSC{Tv,Ti}, ::SparseMatrixCSC{Tv,Ti}, ::AbstractArray{Tp,1}, ::AbstractArray{Tq,1})
-SparseArrays.transpose!
-SparseArrays.adjoint!
 SparseArrays.halfperm!
 SparseArrays.ftranspose!
 SparseArrays.fkeep!

